docs/DeveloperGuide.adoc

```diff
@@ -554,6 +554,62 @@ Cell Formatting
 ** """yummy"", ""juicy"""
 ** """fresh"", ""clean"""
 
+=== Budget Feature
+The `Budget` component has two core feature:
+
+1. Allowing the user to set the monthly budget
+2. Calculating the remanding budget after all the spending in the same month.
+
+The budget component allows the users to modify the current monthly budget `budget AMOUNT` command.
```

Comment:
what is AMOUNT? the input scope should be defined

Reply:
so something like, `budget amount` command, where amount is the desired monthly budget?

Reply:
Is amount an integer, etc